lib/App/Rakubrew/Build.pm

```diff
@@ -186,19 +186,23 @@ sub build_triple {
 }
 
 sub build_zef {
-    my $version = shift;
-
-    _check_git();
-
-    chdir catdir($versions_dir, $version);
-    unless (-d 'zef') {
-        run "$GIT clone $git_repos{zef}";
+  my $version = shift;
```

Comment:
Can you change to 4 space indentation? Then it matches the rest of the rakubrew code.